### ISSUE TITLE
Pixelated maps

### ISSUE THREAD

**Issue by fahlout** [Esri/arcgis-runtime-samples-ios#183]
As in the version before the beta all the maps are still pixellated on screen, which is such a shame. It's even worse in the beta than it is in the current production version. How come nobody has noticed that yet?

**Comment by dlednik** [comment]
@fahlout this was addressed in quartz beta release. Please use Aesthetics mode.

```Swift
self.map.loadWithCompletion { (error) in
    if error == nil {
        let lay = self.map.basemap?.baseLayers[0] as! AGSImageTiledLayer
                lay.renderMode = .Aesthetics
    }
}
```

**Comment by dg0yal** [comment]
There was a temporary regression in the quality of raster tiles in the early betas, but it should be fixed in the final 100.0.0 release of the SDK. Please let us know if you're still seeing the problem, and provide a screenshot or two. Having said that, there is only so much quality that can be offered by tiles that aren't specifically generated for retina displays. The best quality can be achieved through the new vector tile format the the SDK supports.
